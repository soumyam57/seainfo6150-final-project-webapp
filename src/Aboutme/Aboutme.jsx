import React from 'react';
import styles from "./Aboutme.module.css";
import sm from "../images/sm.jpg";
import { Link } from "react-router-dom";

const Aboutme = () => {
    return (
        <div className={styles.flex}>
            <div className={styles.grid}>
                <img className={styles.image} src={sm} alt=""/>
                <div className={styles.div}>
                    <h1>About Me</h1><br /><br />
                    <p>I am Soumya and I am delighted to meet you today</p><br />
                    <p>I hope you liked my website. This website was built using React, Javascript, HTML and CSS.
                        This is Web Design project and the course is instructed by Prof. April Bingham and Teaching Assistant Xinxin
                    </p><br />
                    <p>I would love to know your experience visiting my webiste. Click <Link className={styles.text} to="/feedback">here</Link> to provide your feedback</p><br /><br />
                    <h1>Have Questions?</h1><br />
                    <p>You can reach me at 123-456-7890</p><br /><br />
                    <h2>Thank you for stopping by, have a great day</h2><br />
                </div>
            </div>
        </div>
    )
}

export default Aboutme;