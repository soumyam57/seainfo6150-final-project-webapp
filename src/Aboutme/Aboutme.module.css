.flex{
    background-image: linear-gradient(to right,rgb(83, 15, 15), rgb(49, 12, 25) 70%);
    display: flex;
    flex-direction: row;
    padding: 5%;
}

.grid{
    display: grid;
    grid-template-columns: repeat(2,1fr);
    font-family: 'Arima Madurai', cursive;
    font-size: smaller;
}

.div{
    padding: 2%;
    border-radius: 2%;
    display: flex;
    flex-direction: column;
    box-shadow:0px 0px 10px 5px grey;
    border: 5px solid maroon;
    height: 100%;
    color: rgb(221, 193, 193);
}

.image{
    padding: 2%;
    border-radius: 50%;
    box-shadow:0px 0px 10px 5px grey;
    border: 5px solid maroon;
    height: 100%;
    width: 70%;
}