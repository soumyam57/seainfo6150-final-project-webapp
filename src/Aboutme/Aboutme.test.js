import React from "react";
import { render } from "@testing-library/react";
import Aboutme from "./Aboutme.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders the Aboutme component correctly", () => {
  const { container } = render(
    <BrowserRouter>  
        <Aboutme/>
    </BrowserRouter> 
  );
  expect(container).toMatchSnapshot();
});