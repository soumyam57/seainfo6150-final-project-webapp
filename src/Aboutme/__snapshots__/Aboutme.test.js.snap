// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Aboutme component correctly 1`] = `
<div>
  <div
    class="mainpage"
  >
    <div
      class="grid"
    >
      <div
        class="flex"
      >
        <div
          class="flexitems"
        >
          <img
            alt=""
            class="image"
            src="sm.jpg"
          />
        </div>
        <div
          class="gap"
        >
           
        </div>
        <div
          class="flexitems"
        >
          <h1>
            About Me
          </h1>
          <br />
          <br />
          <p>
            I am Soumya and I am delighted to meet you today
          </p>
          <br />
          <p>
            I hope you liked my website. This website was built using React, Javascript, HTML and CSS. This is Web Design project and the course is instructed by Prof. April Bingham and Teaching Assistant Xinxin
          </p>
          <br />
          <p>
            I would love to know your experience visiting my webiste. Click 
            <a
              class="text"
              href="/feedback"
            >
              here
            </a>
             to provide your feedback
          </p>
          <br />
          <br />
          <h2>
            Have Questions?
          </h2>
          <br />
          <p>
            You can reach me at 123-456-7890
          </p>
          <br />
          <br />
          <h2>
            Thank you for stopping by, have a great day
          </h2>
          <br />
        </div>
      </div>
    </div>
  </div>
</div>
`;
